src/controllers/zendeskController.ts

```diff
@@ -34,6 +32,11 @@ import {
 import TokenService from "../../src/services/tokenService";
 import { withUserFromRequest } from "../types/user";
 
+const isProfileWithValidEmailAddress = (userProfile =>
+  userProfile.email && userProfile.is_email_validated) as (
```

Comment:
Why the need for manual casting? Is explicit typings (i.e. annotate types on function signature) a problem? If so, a possible flaw may be hiding

Reply:
Yeah, you're right. I had a problem with annotating the signature, but I solved it.

Reply:
fixed in https://github.com/pagopa/io-backend/pull/847/commits/876ee601c6aee0f9492c0c94394997165d67af3b